apko/private/util.bzl

```diff
@@ -0,0 +1,119 @@
+"utility functions"
```

Comment:
do they belong in aspect_bazel_lib? pretty easy to just do that now instead of waiting

Reply:
I think yes. I'll file an issue to move them there.

Reply:
https://github.com/chainguard-dev/rules_apko/issues/4